tests/modules/NotificationTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ *
+ * NotificationTest.php is unit test file for Notification module (modules/Notification.php).
+ *
+ */
+namespace Tests\FirstAide;
+
+require_once str_replace('tests/modules', '', dirname(__FILE__)).'/includes/settings.php';
+
+/**
+ * Importing required modules/wrappers
+ */
+use FirstAide\Notification;
+use PHPUnit\Framework\TestCase;
+
+class NotificationTest extends TestCase
+{
+    /**
+     * Method : testSendSms
+     * Description : test to check of the response for sending a sms is a success
+     */
+    public function testSendSms()
+    {
+        $notification = new Notification(Notification::USE_TEST_CREDENTIALS);
+        $sendNotificationRespose = $notification->sendSms('+910000000000', 'Test message');
+        $this->assertTrue($sendNotificationRespose['response']);
```

Comment:
In addition to checking if there is a response, asserts needs to be added to check if the expected response is received in positive and negative case.